crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -5424,7 +5482,7 @@ mod tests {
                     return 42
 
             class Iterable:
-                def __iter__(self) -> Iterator:
+                def __iter__(self) -> Iterator: ...
```

Comment:
oh, great catch! I think now that the accidental syntax error is fixed here, it would now be trivial to port this to an mdtest.